graphql/resolvers.go

```diff
@@ -2628,19 +2628,37 @@ func (r *taskResolver) GeneratedByName(ctx context.Context, obj *restModel.APITa
 }
 
 func (r *taskResolver) IsPerfPluginEnabled(ctx context.Context, obj *restModel.APITask) (bool, error) {
-	var perfPlugin *plugin.PerfPlugin
-	pRef, err := r.sc.FindProjectById(*obj.ProjectId, false)
+	flags, err := evergreen.GetServiceFlags()
 	if err != nil {
 		return false, err
 	}
-	if perfPluginSettings, exists := evergreen.GetEnvironment().Settings().Plugins[perfPlugin.Name()]; exists {
-		err := mapstructure.Decode(perfPluginSettings, &perfPlugin)
+	if flags.PluginAdminPageDisabled {
+		lastGoodVersion, err := model.FindVersionByLastKnownGoodConfig(*obj.ProjectId, -1)
+		if err == nil && lastGoodVersion != nil {
+			parserProject, err := model.ParserProjectFindOneById(lastGoodVersion.Id)
+			if err == nil && parserProject != nil && *parserProject.PerfEnabled {
```

Comment:
If PerfEnabled is explicitly set to false, we should return false, not check the ProjectRef. 
Can just return `utility.FromBoolPtr(parserProject.PerfEnabled)` in the not nil case (we typically prefer this over explicitly dereferencing in the function; Resolvers get away with it because of how they can require obj fields to be populated, and here you _did_ do a nil check but it's still not generally preferred)